pkg/services/rpcsrv/server.go

```diff
@@ -867,6 +867,14 @@ func (s *Server) getVersion(_ params.Params) (any, *neorpc.Error) {
 		}
 		hfs[cfgHf] = height
 	}
+	standbyCommittee, err := keys.NewPublicKeysFromStrings(cfg.StandbyCommittee)
+	if err != nil {
+		return nil, neorpc.NewInternalServerError(fmt.Sprintf("cannot fetch standbyCommittee: %s", err))
+	}
+	committeeKeys, err := standbyCommittee.Convert()
+	if err != nil {
+		return nil, neorpc.NewInternalServerError(fmt.Sprintf("cannot convert standbyCommittee: %s", err))
+	}
```

Comment:
@roman-khimov, are you sure that we need the list of `crypto.PublicKey`? I'd say that the list of `*keys.PublicKey` or `keys.PublicKeys` is enough for our purpose, it allows to avoid this new `Convert` helper and complicated marshalling-related code, whereas from the user-side there's almost no difference.

Reply:
`keys.PublicKey` is exactly what was meant in https://github.com/nspcc-dev/neo-go/pull/3540#discussion_r1698031516 (yeah, `crypto.PublicKey` is an error there because I remember we have `pkg/crypto`) . We almost never use standard `crypto` things.